src/NUnitFramework/framework/Interfaces/ResultState.cs

```diff
@@ -134,6 +134,11 @@ public ResultState(TestStatus status, string label, FailureSite site)
         /// </summary>
         public readonly static ResultState ChildFailure = ResultState.Failure.WithSite(FailureSite.Child);
 
+        /// <summary>
+        /// A suite failed because one or more child tests failed or had errors
```

Comment:
Shouldn't we change this comment, so that it mentions warning? E.g. 
`/// A suite failed because one or more child tests had warnings`
Currently, it is just the same comment as for `ChildFailure`

Reply:
Fixed